decidim-forms/app/models/decidim/forms/question.rb

```diff
@@ -49,6 +50,7 @@ class Question < Forms::ApplicationRecord
       validates :question_type, inclusion: { in: TYPES }
 
       scope :not_separator, -> { where.not(question_type: SEPARATOR_TYPE) }
+      scope :not_title_and_description, -> { where.not(question_type: TITLE_AND_DESCRIPTION_TYPE) }
```

Comment:
I think that instead of adding a new scope (`.not_title_and_description`) we should change this `.not_separator` to something that sums both cases (`.not_informational` or something like that, I'm not so sure about the naming).

The reasoning is that we don't have any use cases at the moment (or at least I don't see any) where we would want the `.not_title_and_description` but we would want the separators -- and the other way also (`.not_separators` but we would want the `title and descriptions` questions). I see that the queries are always `.not_separator.not_title_and_description`. What do you think?

Reply:
Hi @andreslucena 

The title and description must be included in the exports, that's why it is separated from the "separator" type.
I agree we could call the "not_title_and_description" scope "not_informational" to open for future usage and be more explicit.

Reply:
> The title and description must be included in the exports, that's why it is separated from the "separator" type.

Oh, I didn't catch that one. In that case, if there's a good reason, I prefer to leave it as it is. `not_informational` is a new concept, and I just don't see it clear.